compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -168,15 +168,15 @@ public boolean checkForUniqueSymbol(DiagnosticPos pos, SymbolEnv env, BSymbol sy
         }
 
         // if a symbol is found, then check whether it is unique
-        return isSameSymbol(pos, symbol, foundSym);
+        return !isDuplicateSymbol(pos, symbol, foundSym);
```

Comment:
All the usage of `isDuplicateSymbol` starts with a `!` hence, Consider renaming this.

Reply:
renamed and inverted to isDistinctSymbol()